kde-apps/libkdegames/libkdegames_kf6-25.04.0.recipe

```diff
@@ -6,13 +6,13 @@ COPYRIGHT="2010-2025 KDE Organisation"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/libkdegames-$portVersion.tar.xz"
-CHECKSUM_SHA256="d640d5a590450e476e903a341e916004a46ad72d0db07f4201e75e4e5a7c02e5"
+CHECKSUM_SHA256="69a21bae741b28e8c79c2dfdee352a7cebc9c6658aaa58388d581b2a7a40047f"
 SOURCE_DIR="libkdegames-$portVersion"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
-libVersion="6.0.241203"
+libVersion="6.0.${portVersion//./}0"
```

Comment:
I guess this will break with the next patch update, if the pattern is the same as before. For example, what you wrote would have been `241230` for the old version. This only works if the last part is 0.

Reply:
Looking at the output it does look right? eg `libVersion="6.0.240500"`

Reply:
From buildmaster:
```
provides {
       libkdegames_kf6_devel = 25.04.0
       devel:libkdegames6 = 6.0.250400 compat >= 6
       devel:libkdegames6private = 6.0.250400 compat >= 6
}
```

Reply:
LOL, I just got what you meant :) for 25.04.1*

Reply:
Yes, if the last part is 0, it is correct (00 is the same as 00), but if it 1 after the next update, this will be `10` and not `01`.